src/components/Map/map.js

```diff
@@ -317,45 +317,69 @@ class Map extends Component {
       // the first render of this component, so the marker is not shown until
       // the user starts interacting with the app.
       return;
+    } else {
+      this.updateZoom(distance);
```

Comment:
nit: you can move this out of the else clause because of the return statement in the if.